routes/rider.js

```diff
@@ -1,10 +1,11 @@
 var express = require('express')
 var router = express.Router()
 var db = require('../model/db.js')
+const connect = require('connect-ensure-login')
 
 /* GET user-rider's rides/bids listing page. */
-router.get('/', async function(req, res, next) {
-  const riderId = req.query.user_id_field
+router.get('/', connect.ensureLoggedIn('/login') ,async function(req, res, next) {
```

Comment:
Is it possible to remove the /login here since the other routers don't have them?

Reply:
This is ok, connect.ensureLoggedIn('/login') redirects to `domain/login` which is the login page. it does not redirect to `domain/route/login`